integrations/openshift.md

```diff
@@ -39,585 +39,77 @@ Here's a preview of the Kubernetes Pods dashboard:
 
-### Install and Configure Wavefront Operator on OpenShift Enterprise 4.x
+### Install and Configure Wavefront Helm Chart on OpenShift Enterprise 4.x
 
-This page contains the Installation and Configuration steps for full-stack monitoring of OpenShift clusters using Wavefront Operator.
+This page contains the Installation and Configuration steps for full-stack monitoring of OpenShift clusters using Wavefront Helm Chart.
 
-#### Install and Configure Wavefront Operator 
+#### Install and Configure Wavefront Helm Chart
 
-1.  Log in to OpenShift Web UI as administrator.
-2.  Create a project with name `wavefront`.
-3.  In the Left pane, navigate to **Catalog** > **OperatorHub**.
-4.  From the list of Operator types select **Monitoring** > **Wavefront**.
-5.  Click on the **Wavefront Operator** and click **Install**.
-6.  Select `wavefront` as namespace to subscribe to.
-7.  When the subscription is successful, the operator is listed under **Installed Operators** and deploys Wavefront Proxy and Wavefront Collector CRD's into the project.
-8.  Select **Installed Operators** > **Wavefront Operator** > **Wavefront Proxy** > **Create New**  to deploy the proxy.
-9.  Create Wavefront Proxy custom resource by specifying the following parameters in the proxy spec and leave rest of the values as defaults.
-    * token → YOUR_API_TOKEN
-    * url → https://YOUR_CLUSTER.wavefront.com
-10. Click **Create**.  This deploys the proxy service named `example-wavefrontproxy` with port 2878 as metric port.  In addition, Operator creates Persistent Volume Claim (PVC) with the name `wavefront-proxy-storage` using default underlying Persistent Volume (PV).
-11. Select **Installed Operators** > **Wavefront Operator** > **Wavefront Collector** > **Create New** to deploy the collector.
-12. Click **Create** without changing any values in the proxy definition.
+1. Log in to OpenShift Web UI as administrator.
+2. Create a project with name `wavefront`.
+3. In the Left pane, navigate to **Helm**, select **Install a Helm Chart from the developer catalog**.
+4. Search for **Wavefront** and click **Install Helm Chart**.
+5. Install from **form view** or **yaml view**. Replace the following with your values:
+    * clusterName → <KUBERNETES_CLUSTER_NAME>
+    * token → <YOUR_API_TOKEN>
+    * url → https://<YOUR_CLUSTER>.wavefront.com
+6. Click **Install**.
 
-Because default parameters are used, the collector runs as a daemonset and uses `example-wavefrontproxy` as sink. The collector auto-discovers the pods and services that expose metrics and dynamically starts collecting metrics for the targets. It collects metrics from the kubernetes API server if configured.
+Because default parameters are used, the collector runs as a daemonset and uses `wavefront-proxy` as sink. The collector auto-discovers the pods and services that expose metrics and dynamically starts collecting metrics for the targets. It collects metrics from the kubernetes API server if configured.
```

Comment:
...the Collector...
RK>>we're using "Collector" and not "collector" elsewhere on this page. I'm not sure I like that, but let's be consistent.